content/yaml/building-a-react-native-app.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Building a React Native app
+description: Building a React Native app with YAML.
+weight: 5
+---
+
+React Native is a cross-platform solution that allows you to build apps for both iOS and Android faster using a single language. When working with YAML, the basics are still the same, the build scripts are added to the `scripts` section in the [overall architecture](../yaml/yaml/#template).
+
+## Android
+
+Script for building an Android application:
+
+    - echo "sdk.dir=$HOME/programs/android-sdk-macosx" > "$FCI_BUILD_DIR/android/local.properties"
+    - cd android && ./gradlew bundleDebug
```

Comment:
again, is it bundleDebug or just build? @BondarenkoStas

Reply:
answered here https://github.com/codemagic-ci-cd/codemagic-docs/pull/102#discussion_r448984340